public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -165,9 +166,9 @@ export const LogRowMessage = memo((props: Props) => {
       }
       <td className={styles.logsRowMessage}>
         <div className={wrapLogMessage ? styles.positionRelative : styles.horizontalScroll}>
-          <button className={`${styles.logLine} ${styles.positionRelative}`}>
```

Comment:
With these changes you can clean up some of the extra styles we're setting to override the button defaults, such as border, background color, and probably user select too. https://github.com/grafana/grafana/blob/main/public/app/features/logs/components/getLogRowStyles.ts#L290-L291